soc/arm/nuvoton_npcx/Kconfig

```diff
@@ -20,6 +20,21 @@ menuconfig NPCX_HEADER
 
 if NPCX_HEADER
 
+config NPCX_IMAGE_OUTPUT_BIN
```

Comment:
why don't you just use the generic option BUILD_OUTPUT_BIN defined in Kconfig.zephyr?

Reply:
The npcx hex binary is generated from an npcx raw binary. I would like to add a dependency checking in Kconfig, so I add an option here. Moreover, currently, some applications that image need firmware header, and some applications don't need the firmware header. I think adding options will more clear.